upgrades/upgrade_testing_framework/steps/step_post_upgrade_robot_test.py

```diff
@@ -0,0 +1,22 @@
+from upgrade_testing_framework.core.framework_step import FrameworkStep
+from robot_tests.test_executor import TestExecutor
+
+class PostUpgradeRobotTest(FrameworkStep):
```

Comment:
Same comment; how about `PerformPostUpgradeTest` instead?  Not sure we need to include "Robot" in the step name.

Reply:
Agreed I was on the fence about adding robot myself, will modify